src/AgeComparator.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Comparator;
+
+public class AgeComparator implements Comparator<Person> {
+
+    //The second comparator sorts all people in descending order by their age
+
+    @Override
+    public int compare(Person person1, Person person2) {
+        if (person1.getAge() > person2.getAge()) {
+            return -1;
+        } else if (person1.getAge() < person2.getAge()) {
```

Comment:
first if and else if are redundant, if you read the comparteTo documentation it states:

Compares two Integer objects numerically.
Params:
anotherInteger – the Integer to be compared.
Returns:
the value 0 if this Integer is equal to the argument Integer; a value less than 0 if this Integer is numerically less than the argument Integer; and a value greater than 0 if this Integer is numerically greater than the argument Integer (signed comparison).